examples/readtableintoStruct.rs

```diff
@@ -0,0 +1,127 @@
+
+extern crate cdrs;
+use cdrs::client::CDRS;
+use cdrs::authenticators::PasswordAuthenticator;
+use cdrs::consistency::Consistency;
+use cdrs::types::CBytes;
+use cdrs::compression::Compression;
+use cdrs::types::value::Value;
+use cdrs::types::rows::Row;
+use cdrs::types::IntoRustByName;
+
+/// this example is to pull employee records from emp table
+///
+/// # Examples
+///
+/// ```
+///CREATE KEYSPACE my_namespace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
+
+/// CREATE TABLE emp(
+/// emp_id int PRIMARY KEY,
+/// emp_name text,
+/// emp_city text,
+/// emp_sal varint,
+/// emp_phone varint
+/// );
+
+
+/// INSERT INTO emp (emp_id, emp_name, emp_city,emp_phone, emp_sal) VALUES(1,'alex', 'NJ', 9848022338, 50000)
+/// INSERT INTO emp (emp_id, emp_name, emp_city,emp_phone, emp_sal) VALUES(2,'harry', 'FLL', 9848022338, 50000)
+///
+/// 
+/// ```
+#[derive(Debug,Default)]
+struct Employee {
+    pub id: i64,
+    pub emp_name: String,
+    pub emp_city: String,
+    pub emp_sal: i64,
+    pub emp_phone: i64,
+}
+
+fn main() {
+
+    let authenticator = PasswordAuthenticator::new("user", "pass");
+    let addr = "127.0.0.1:9042";
+
+    // pass authenticator into CDRS' constructor
+    let client = CDRS::new(addr, authenticator).unwrap();
+
+    // start session without compression
+    let select_query = String::from("SELECT * FROM my_namespace.emp;");
+    // Query parameters:
+    let consistency = Consistency::One;
+    let values: Option<Vec<Value>> = None;
+    let with_names: Option<bool> = None;
+    let page_size: Option<i32> = None;
+    let paging_state: Option<CBytes> = None;
+    let serial_consistency: Option<Consistency> = None;
+    let timestamp: Option<i64> = None;
+
+    let use_query = String::from("USE my_namespace;");
+    match client.start(Compression::None) {
+        Ok(session) => {
+            match session.prepare(use_query) {
```

Comment:
It may be not convenient to use current prepare + execute flow because related API is not developed well. So I would recommend to use `query` or `query_with_builder`